pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [tool.black]
 line-length = 120
-target-version = ['py27', 'py35', 'py36', 'py37', 'py38']
+target-version = ['py36', 'py37', 'py38']
```

Comment:
please add py39

Reply:
3.9 is not valid yet: `Error: Invalid value for '-t' / '--target-version': invalid choice: 3.9. (choose from py27, py33, py34, py35, py36, py37, py38)`